### PR TITLE
Make aad b2c support global properties

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , @backwind1233 , please help to review this PR when you have time.

File: aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml [hunk 1]
```diff
@@ -15,6 +15,18 @@
...
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>spring-cloud-azure-dependencies</artifactId>
+        <version>4.0.0-beta.3</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```
Comment (saragluna) [inline aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml:29]: @backwind1233 should this dependencyManagement be in the parent pom file or in each sample's pom?

Comment (chenrujun) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml:29]: Parent pom already have dependency management.
We should delete the `dependencyManagement` here after 4.0.0-beta.3(or 4.0.0) released.

Comment (saragluna) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml:29]: How to make sure that will be done?

Comment (chenrujun) [reply aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml:29]: Issue created: https://github.com/Azure/azure-sdk-for-java/issues/25828

**Pull request merged**